deploy-manage/tools.md

```diff
@@ -1,32 +1,60 @@
 ---
 mapped_pages:
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/high-availability.html
+
+applies:
+  stack: all
+  serverless: all
```

Comment:
please remove serverless from here because none of the tools are available for serverless

```suggestion
```